docs/product/others/websocket-events.md

```diff
@@ -215,6 +215,44 @@ The following payload will be returned for an agent/admin.
 }
 ```
 
+#### Notification
+
+The following payload will be returned for a notification.
+
+```json
+{
+    "id": 795302,
+    "notification_type": "conversation_mention",
```

Comment:
Please fix this payload by replacing the content with the data type. Please check the other object definition for reference.